### PR TITLE
virt-install: update page

### PR DESCRIPTION
**Pull request by luzeal**
Hello, I have added a few examples that I often use, and that may be of interest to other users. Some examples could probably be shortened.

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/virt-install.md [hunk 1]
```diff
@@ -3,6 +3,26 @@
 > Create virtual machines with libvirt and begin OS installation.
 > More information: <https://virt-manager.org/>.
 
-- Create a virtual machine with 1 GiB RAM and 12 GiB storage and start Debian installation:
+- Create a virtual machine with 1 GiB RAM and 12 GiB storage and start a Debian installation:
```
Comment (nicokosi) [inline pages/common/virt-install.md:6]: Not related to this PR but we may fix the unit symbol (see https://en.wikipedia.org/wiki/Gigabyte):
```suggestion
- Create a virtual machine with 1 GB RAM and 12 GB storage and start a Debian installation:
```

File: pages/common/virt-install.md [hunk 2]
```diff
@@ -3,6 +3,26 @@
...
+
+- Create a diskless live virtual machine without an emulated sound device or a USB controller. Don't start an installation and don't autoconnect to console but attach a cdrom to it (might be useful for when using a live CD like tails):
+
+`virt-install --name {{vm_name}} --memory {{512}} --disk {{none}} --controller {{type=usb,model=none}} --sound {{none}} --autoconsole {{none}} --install {{no_install=yes}}  --cdrom {{path/to/tails.iso}}`
+
+- Create a virtual machine with with 16 GiB RAM, 250 GiB storage, 8 cores with hyperthreading, a specific CPU topology, and a CPU model that shares most features with the host CPU:
+
+`virt-install --name {{vm_name}} --cpu {{host-model}},topology.sockets={{1}},topology.cores={{4}},topology.threads={{2}} --memory {{16384}} --disk path={{path/to/image.qcow2}},size={{250}} --cdrom {{path/to/debian.iso}}`
+
+- Create a virtual machine and kickstart an automated deployment based on Fedora 35 using only remote ressources (no ISO required):
```
Comment (nicokosi) [inline pages/common/virt-install.md:22]:
```suggestion
- Create a virtual machine and kickstart an automated deployment based on Fedora 35 using only remote resources (no ISO required):
```

File: pages/common/virt-install.md (hunk 1 above)
Comment (nicokosi) [reply pages/common/virt-install.md:6]: (same comment for units below)

**Comment by nicokosi** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/1522766309) for this PR failed with the following error(s):
> 
> ```
> pages/common/virt-install.md:28:251 MD013/line-length Line length [Expected: 250; Actual: 632]
> pages/common/virt-install.md:22: TLDR014 Page should contain no trailing whitespace
> ```
> 
> Please fix the error(s) and push again.



> The [build](https://github.com/tldr-pages/tldr/actions/runs/1522766309) for this PR failed with the following error(s):
> 
> ```
> pages/common/virt-install.md:28:251 MD013/line-length Line length [Expected: 250; Actual: 632]
> pages/common/virt-install.md:22: TLDR014 Page should contain no trailing whitespace
> ```
> 
> Please fix the error(s) and push again.

🙏

**Comment by nicokosi** [comment]
Thank you for the explanation in the description. 👍

File: pages/common/virt-install.md (hunk 1 above)
Comment (luzeal) [reply pages/common/virt-install.md:6]: Thanks for the correction

File: pages/common/virt-install.md (hunk 2 above)
Comment (luzeal) [reply pages/common/virt-install.md:22]: Likewise

**Review by luzeal — COMMENTED** [review]
I have erased the latest example and did commit a fix with the corrections (I believe ^^)

**Review by nicokosi — APPROVED** [review]
Looks good to me! 👌 (but I don't know anything about `virt-install` 😇)

**Comment by luzeal** [comment]
> Looks good to me! ok_hand (but I don't know anything about `virt-install` innocent)

Thanks !

It is actually a pretty great tool, which I use with kickstart files to automatically deploy operating systems. 

See [here](https://wiki.phyllo.me/gofurther/virt-install#automated-installation-using-a-remote-kickstart-file) for a complete working example (which will deploy a trimmed-down version of Fedora Workstation)

**Comment by nicokosi** [comment]
Any other approving reviews? :innocent:

**Comment by 258204** [comment]
Isn't path an optional parameter? That defaults to either $HOME/.local/share/libvirt/images or in /var/lib/libvirt/images if invoked from root?

**Review by sbrl — APPROVED** [review]
Wow, these descriptions and examples are wildly long. I suspect though that like `whiptail`, there isn't anything we can do about this.

Thanks, @luzeal!

**Comment by sbrl** [comment]
Maintainers: Don't forget to attach the `review needed` to PRs that need a 2nd review :-)

**Comment by luzeal** [comment]
Thanks for the merge! 

> Isn't path an optional parameter? That defaults to either $HOME/.local/share/libvirt/images or in /var/lib/libvirt/images if invoked from root?

That may be the case. Hopefully, someone will pass by and trim these commands further (maybe me in another pull request : )

**Pull request merged**